src/main/java/sysc4806/project24/mini_survey_monkey/models/MultipleChoiceResponse.java

```diff
@@ -0,0 +1,19 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+
+import jakarta.persistence.Entity;
+
+@Entity
+public class MultipleChoiceResponse extends Response {
```

Comment:
Shouldn't this map to a question?

Reply:
it already inherits the question field from the Response class, which is its superclass. The @ManyToOne relationship between Response and Question ensures that a MultipleChoiceResponse is associated with a specific Question, does this make sense ?